stack.yaml

```diff
@@ -44,6 +44,9 @@ extra-deps:
   commit: 563e96238dfe392dccf68d93953c8f30fd53bec8
   subdirs:
     - ki
+# lts 18.28 provides 0.8.2 but we need un-released version that fixes apple m1 bug
+- github: judah/haskeline
+  commit: 7a83e7659211049f3bd167e7a13bc1cb4c42c91b
```

Comment:
According to https://github.com/judah/haskeline/pull/163 (from aug 2021) that PR was included in haskeline 0.8.1.3. So if we are using 0.8.2, shouldn't that be newer / good enough?

What was happening when you didn't include this (older?) unreleased version?